src/SampleUtterances.ts

```diff
@@ -55,11 +55,10 @@ export class SampleUtterances {
     }
 
     public samplesForIntent(intent: string): SamplePhrase [] {
-        let samples: SamplePhrase[] = [];
-        if (intent in this.samples) {
-            samples = this.samples[intent];
+        if (!(intent in this.samples)) {
```

Comment:
Is this fixing a bug? I prefer to not do early returns. Do we have a philosophy on this?

Reply:
Okay, as long as you are applying this across our codebases, I am fine with this approach